recipes/conversation_view/feed_pill/feed_item_pill_default.story.vue

```diff
@@ -1,12 +1,13 @@
 <template>
   <dt-recipe-feed-item-pill
-    :icon-name="$attrs.iconName"
-    :title="$attrs.title"
-    :wrapper-class="$attrs.wrapperClass"
-    :button-class="$attrs.buttonClass"
-    :border-color="$attrs.borderColor"
-    :aria-label="$attrs.ariaLabel"
-    :toggleable="$attrs.toggleable"
+    :icon-name="iconName"
+    :title="title"
+    :wrapper-class="wrapperClass"
+    :button-class="buttonClass"
+    :border-color="borderColor"
+    :aria-label="ariaLabel"
+    :toggleable="toggleable"
+    :default-toggled="defaultToggled"
```

Comment:
This properties need to be `$attrs.*` if they're coming from `.stories.js` on Vue 3 version.